packages/strapi-parts/src/Text/index.js

```diff
@@ -49,14 +49,21 @@ export const Subtitle = styled(Text)`
   line-height: 1.5;
 `;
 
-export const TextButton = styled(Text)`
+export const ButtonText = styled(Text)`
   font-weight: 600;
   line-height: 1.14;
 `;
 
-export const TableLabel = styled(TextButton)`
+export const TableLabel = styled(ButtonText)`
   font-weight: 600;
   font-size: ${11 / 16}rem;
   line-height: 1.45;
   text-transform: uppercase;
 `;
+
+export const EllipsisText = styled(Text)`
```

Comment:
WDYT about having ellipsis as a component and not a prop ?
Some might use a TableLabel, ButtonText, or any other Text component with an ellipsis prop on it

Reply:
I understand that but the problem is that the Text component is a span. To use the ellipsis style, we need a `block` element not an `inline`. I'm afraid that moving the text component to a `block` will break a lot of things in the other components, layouts or migrated views. @bulby97

Reply:
Got your point, but ellipsis can also be applied to a inline-block. I don't think the inline block can break inline stuff, but maybe I'm wrong

Reply:
You are right, I can add the ellipsis to the inline-block my bad.